src/nebula/types.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * Created by Wu Jian Ping on - 2021/06/09.
+ */
+
+import { AsyncResource } from 'async_hooks'
+
+interface ConnectionOption {
+  host: string;
+  port: number;
+  userName: string;
+  password: string;
+  database: string;
+}
+
+interface Endpoint {
+  host: string;
+  port: number
+}
+
+interface ClientOption {
+  // nebula服务器列表
+  servers: string[] | Endpoint[];
+  // 用户名
+  userName: string;
+  // 密码
+  password: string;
+  // 数据库名称
+  database: string;
```

Comment:
It's for space? maybe it's more clear to rename it to space ?

Reply:
@nianiaJR rename `database`  to `space` done